### PR TITLE
set specific stream bound limits

### PR DESCRIPTION
**Pull request by shotasilagadze**
Limiting default `StreamsOutbound` and `StreamsInbound` for p2p resource manager.

### CODE REVIEW

File: cl/sentinel/sentinel.go [hunk 1]
```diff
@@ -242,7 +242,17 @@ func New(
 		return nil, err
 	}
 
-	rmgr, err := rcmgr.NewResourceManager(rcmgr.NewFixedLimiter(rcmgr.DefaultLimits.AutoScale()), rcmgr.WithTraceReporter(str))
+	defaultLimits := rcmgr.DefaultLimits.AutoScale()
+	newLimit := rcmgr.PartialLimitConfig{
+		System: rcmgr.ResourceLimits{
+			// Default:  64 * 16, now it's 64 * 4
+			StreamsOutbound: rcmgr.LimitVal(defaultLimits.ToPartialLimitConfig().System.StreamsOutbound/4),
```
Comment (Giulio2002) [inline cl/sentinel/sentinel.go:249]: i would say to set it to something more conservative - try `16`

Comment (shotasilagadze) [reply cl/sentinel/sentinel.go]: dividing by 16 now but, pay attention to one thing. I'm converting defaultLimits to ToPartialLimitConfig() only to read  `System.StreamOutbound` value. Couldn't find the other way to reach that value. I'll check it later.

Comment (Giulio2002) [reply cl/sentinel/sentinel.go]: I meant literally `StreamsOutbound: 16`

Comment (shotasilagadze) [reply cl/sentinel/sentinel.go]: I think we need to restrict `StreamInbound`. I will set both to 16

**Review by Giulio2002 — APPROVED** [review]

**Pull request merged**